tests/util/ocp/test_common.py

```diff
@@ -76,16 +83,33 @@ def tearDown(self):
 
     def test_get_cluster_id_from_provider(self):
         """Test that the cluster ID is returned from OCP provider."""
-        cluster_id = utils.get_cluster_id_from_provider(self.ocp_test_provider_uuid, self.test_schema)
+        cluster_id = utils.get_cluster_id_from_provider(self.ocp_test_provider_uuid)
         self.assertIsNotNone(cluster_id)
 
     def test_get_cluster_id_from_non_ocp_provider(self):
         """Test that None is returned when getting cluster ID on non-OCP provider."""
-        cluster_id = utils.get_cluster_id_from_provider(self.aws_test_provider_uuid, self.test_schema)
+        cluster_id = utils.get_cluster_id_from_provider(self.aws_test_provider_uuid)
         self.assertIsNone(cluster_id)
 
-    def test_get_multiple_clusters_one_ocp_provider(self):
-        """Test that Exception is raised for multiple cluster IDs on a single OCP provider."""
-        self.creator.create_ocp_report_period(provider_id=self.provider_id)
-        with self.assertRaises(MasuConfigurationError):
-            utils.get_cluster_id_from_provider(self.ocp_test_provider_uuid, self.test_schema)
+    def test_get_provider_uuid_from_cluster_id(self):
+        """Test that the provider uuid is returned for a cluster ID."""
+        cluster_id = self.ocp_provider_resource_name
+        provider_uuid = utils.get_provider_uuid_from_cluster_id(cluster_id)
+        self.assertIsNotNone(provider_uuid)
```

Comment:
It would be nice to do some basic type-checking in these unit tests. This should be a valid uuid, not just a non-None value. An empty list or dict would pass this test.

Reply:
Sounds good. I'll add that